WBooks/Books Table/BooksViewModel.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  BooksViewModel.swift
+//  WBooks
+//
+//  Created by Carolina Arcos on 8/24/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+import ReactiveCocoa
+import ReactiveSwift
+import Result
+
+class BooksViewModel {
+    
+    // MARK: - Properties
+    
+    private let mutableBooks = MutableProperty([Book]())
+    private let _error: Signal<NSError, NoError>.Observer
+    let books: Property<[Book]>
+    let errorSignal: Signal<NSError, NoError>
+    let getBooks: () -> SignalProducer<[Book], NSError>
+    
+    // MARK: - Initializers
+    
+    init(getBooks: @escaping () -> SignalProducer<[Book], NSError>) {
+        self.getBooks = getBooks
+        books = Property(mutableBooks)
+        (errorSignal, _error) = Signal<NSError, NoError>.pipe()
+        loadBooks()
```

Comment:
This shouldn't be called in the init, it should be call in the controller when it is time for it.